bun-tests/fake-snippets-api/fixtures/get-test-server.ts

```diff
@@ -76,29 +76,48 @@ const seedDatabase = (db: DbClient) => {
       phone: "555-123-4567",
     },
   })
+
   const account2 = db.addAccount({
     github_username: "jane",
   })
+
+  const circuit_json = [
+    {
+      type: "source_component",
+      ftype: "simple_resistor",
+      source_component_id: "source_component_1",
+      name: "R1",
+      resistance: "1k",
+    },
+  ]
+
   const order = db.addOrder({
     account_id: account.account_id,
     is_running: false,
     is_started: false,
     is_finished: false,
     error: null,
     has_error: false,
-    circuit_json: [
-      {
-        type: "source_component",
-        ftype: "simple_resistor",
-        source_component_id: "source_component_1",
-        name: "R1",
-        resistance: "1k",
-      },
-    ],
+    circuit_json,
     created_at: new Date().toISOString(),
     started_at: null,
     completed_at: null,
   })
 
-  return { account, order, account2 }
+  const packageRelease = db.addPackageRelease({
+    package_id: "pkg_1",
+    version: "0.0.1",
+    created_at: new Date().toISOString(),
+    is_latest: true,
+    is_locked: false,
+  })
+
+  const orderQuote = db.addOrderQuote({
+    account_id: account.account_id,
+    package_release_id: packageRelease?.package_release_id,
+    vendor_name: "jlcpcb",
+    circuit_json: JSON.stringify(circuit_json),
```

Comment:
an order quote will never have both of these, it either has a package release id or circuit_json

Reply:
all of this code should be in `seedDatabase`